client/my-sites/domains/controller.jsx

```diff
@@ -256,7 +256,14 @@ const jetpackNoDomainsWarning = ( context, next ) => {
 					path={ context.path.indexOf( '/domains/add' ) === 0 ? '/domains/add' : '/domains/manage' }
 					title="My Sites > Domains > No Domains On Jetpack"
 				/>
-				<JetpackManageErrorPage template="noDomainsOnJetpack" siteId={ selectedSite.ID } />
+				<EmptyContent
+					title={ translate( 'Domains are not available for this site.' ) }
+					line={ translate(
+						'You can only purchase domains for sites hosted on WordPress.com at this time.'
+					) }
```

Comment:
This is a completely non-blocking suggestion, but I feel like our design-minded folks might appreciate some widow prevention in these two lines of copy:

```suggestion
					title={ preventWidows( translate( 'Domains are not available for this site.' ) ) }
					line={ preventWidows( translate(
						'You can only purchase domains for sites hosted on WordPress.com at this time.'
					) ) }
```

Example of what this would resolve (note the single word at the end of one line in each screenshot):

<img height="225" alt="image" src="https://user-images.githubusercontent.com/670067/124766874-ec267400-defc-11eb-8a34-dc4d6f0d7c0f.png"> <img height="225" alt="image" src="https://user-images.githubusercontent.com/670067/124766791-d87b0d80-defc-11eb-945d-194fbc5cc7b2.png">

Reply:
I'll pass on this if you don't mind. From the screenshots you provided it seems like when we fix it for the title, we break it for the paragraph below 😉